magellan-library/src/main/java/com/wealthfront/magellan/core/Journey.kt

```diff
@@ -11,5 +13,5 @@ public abstract class Journey<V : ViewBinding>(
   getContainer: V.() -> ScreenContainer
 ) : Step<V>(inflateBinding) {
 
-  protected var navigator: LinearNavigator by attachFieldToLifecycle(LinearNavigator { viewBinding!!.getContainer() })
+  @VisibleForTesting(otherwise = PROTECTED) public var navigator: LinearNavigator by attachFieldToLifecycle(LinearNavigator { viewBinding!!.getContainer() })
```

Comment:
nice! 

Random but in case you didn't know, there also something called RestrictTo: https://developer.android.com/studio/write/annotations#restrict. I learned about that somewhat recently so thought I'd mention it

Reply:
Ooh good to know. In this case, we want it to be `protected | tests`, so I think `@VisibleForTesting` is what we want. But good to have another tool in the toolbox!